extra-utils/swaybg/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=swaybg
+PKGSEC=x11
+PKGDEP="wayland gdk-pixbuf"
+BUILDDEP="wayland-protocols scdoc fish"
+PKGDES="Wallpaper tool for Wayland compositors"
```

Comment:
Wallpaper manager, wallpaper setter, wallpaper previewer? What exactly is it?

Reply:
Wallpaper setter. I'll change the description.